packages/dom-expressions/src/server.js

```diff
@@ -1,7 +1,19 @@
 import { Aliases, BooleanAttributes, ChildProperties } from "./constants";
 import { sharedConfig, root } from "rxcore";
 import { createSerializer, getLocalHeaderScript } from "./serializer";
-export { createComponent } from "rxcore";
+
+export { getOwner, createComponent, effect, memo, untrack } from "rxcore";
```

Comment:
Is this list accurate? I based this off of `web/src/core.ts`, which I assume indicates which APIs are available server-side, but I did not include those that were already included here in `server.js`. This brings server.js exports to closer parity with client.js exports.

Reply:
Ok, no, not accurate. The code is difficult to follow.

I see that the build output has code injected, that I need to avoid clashing with:

```js
const isServer = true;
const isDev = false;
function render() {}
function hydrate() {}
function insert() {}
function spread() {}
function addEventListener() {}
function delegateEvents() {}
function Dynamic(props) {
  const [p, others] = splitProps(props, ["component"]);
  const comp = p.component,
    t = typeof comp;
  if (comp) {
    if (t === "function") return comp(others);else if (t === "string") {
      return ssrElement(comp, others, undefined, true);
    }
  }
}
function Portal(props) {
  return "";
}
```

Reply:
Where is this list of exports? For example `function render() {}` is injected, and that is not exported in `server.d.ts`.

Is `babel-plugin-transform-rename-import` doing the injection?

Reply:
it is confusing because it looks like the exports from `dom-expressions` get merged with the exports from `web/src/index.ts` into a single file. It bespoke and totally non-obvious how the code comes together.